sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -600,6 +600,7 @@ if (contextVersion > 1) {
   shouldBe('gl.getUniform(samplerForWebGL2Program, s3DValLoc)', '0');
   shouldBe('gl.getUniform(samplerForWebGL2Program, s2DArrayValLoc)', '1');
 }
+wtu.glErrorShouldBe(gl, gl.NO_ERROR);
```

Comment:
Add this line to show the error in getUniform, and prevent the errors fall to getVertexAttrib.